prow/plugins/golint/golint.go

```diff
@@ -139,30 +139,49 @@ func newProblems(cs []github.ReviewComment, ps map[string]map[int]lint.Problem)
 
 // problemsInFiles runs golint on the files. It returns a map from the file to
 // a map from the line in the patch to the problem.
-func problemsInFiles(r *git.Repo, files map[string]string) (map[string]map[int]lint.Problem, error) {
+func problemsInFiles(r *git.Repo, files map[string]string) (map[string]map[int]lint.Problem, github.DraftReviewComment, error) {
 	problems := make(map[string]map[int]lint.Problem)
+	var lintErrorComment github.DraftReviewComment
 	l := new(lint.Linter)
 	for f, patch := range files {
 		problems[f] = make(map[int]lint.Problem)
 		src, err := ioutil.ReadFile(filepath.Join(r.Dir, f))
 		if err != nil {
-			return nil, err
+			return nil, lintErrorComment, err
 		}
 		ps, err := l.Lint(f, src)
 		if err != nil {
-			return nil, fmt.Errorf("linting %s: %v", f, err)
+			// Get error line by parsing the error message
+			errLineIndexStart := strings.LastIndex(err.Error(), f) + len(f)
+			reNumber := regexp.MustCompile(`:([0-9]+):`)
+			matches := reNumber.FindStringSubmatch(err.Error()[errLineIndexStart:])
+			if len(matches) < 2 {
+				return nil, lintErrorComment, err
+			}
+			errLineString := matches[1]
+			errLine, errAtoi := strconv.Atoi(errLineString)
+			if errAtoi != nil {
+				return nil, lintErrorComment, err
+			}
+			// Create DraftReviewComment for lint error
+			lintErrorComment = github.DraftReviewComment{
+				Path:     f,
+				Position: errLine,
+				Body:     fmt.Sprintf("linting %s: %v", f, err),
```

Comment:
We don't need to mention the file since this is a review comment on the file.
The full `err` message is probably excessive too. e.g. 
```
prow/plugins/golint/suggestion/testdata/range.go:22:1: expected declaration, found 'for'
```
Only really needs `expected declaration, found 'for'` since the rest is implied from the comment path and position.